modin/engines/ray/generic/io.py

```diff
@@ -103,6 +103,8 @@ class RayIO(BaseIO):
     #
     # Signature: (num_splits, sql, con, index_col, kwargs)
 
+    from pyarrow.parquet import ParquetFile
```

Comment:
This import should be moved inside the relevant functions/methods that require it.